lib/ood_core/job/adapters/sge/qstat_xml_j_r_listener.rb

```diff
@@ -153,6 +168,17 @@ def end_JAT_task_number
     @running_tasks << @current_text
   end
 
+  def end_PN_path
+    if (@stdout_PN_path)
+      @stdout_PN_path = false
+      @parsed_job[:native][:PN_path] = @current_text
+    end
+  end
+
+  def end_ST_name
+    @parsed_job[:native][:ST_name] = @parsed_job[:native][:ST_name] + @current_text + ' '
+  end
```

Comment:
What's `ST_name` supposed to for us, the output file?  If you see the [UGE xml in another PR](https://github.com/OSC/ood_core/blob/c2b64e6e39c300931530983afebd47f10b4bec25/spec/job/adapters/sge/output_examples/uge_qstat_jr.xml), it looks like it's an element for the submission command?  I can't find an example of it in our current test output files.

Either way, there's no test case for it, and I'm not sure if it would be end up being what you expect. (I imagine parsers take sets of ST_name, ST_pos and ST_access_specifier to rebuild the command).

Reply:
OK I see this is additive, continually appending.

Reply:
The discourse page mentioned uses ST_name and it seems like the submission_args which is something shown in active jobs.